addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -207,7 +207,7 @@
                                     <field name="stage_id" domain="[('user_id', '=', False), ('project_ids', 'in', [project_id])]"/>
                                     <button name="action_open_task" type="object" title="View Task" string="View Task" class="btn btn-link float-end"
                                             context="{'form_view_ref': 'project.project_sharing_project_task_view_form', 'search_view_ref': 'project.project_sharing_project_task_view_search'}"
-                                            invisible="project_id or project_id != id"/>
+                                            invisible="not project_id and project_id != id"/>
```

Comment:
The condition now seems a bit weird, IMO. 🤔

Reply:
> The condition now seems a bit weird, IMO. 🤔

Here, the project ID is compared with task ID.

I've now updated the code. If the subtask is unsaved, then the view button is invisible.

thanks